### ISSUE TITLE
Adding array of where's

### ISSUE THREAD

**Issue by calebn** [saintsystems/odata-client-php#21]
When trying to add an array of where filters (which I had working in a previous version) I get an extra = at the beginning of the items in the filter. Here is the generated url:
`https://api.channeladvisor.com/v1/Products?$select=Sku,ID&$filter=(=Sku%20eq%20'MCN-GATRTDB-20X3-PINK'%20or%20Sku%20eq%20'MCN-RDJASBD12OZ-12OZ'%20or%20Sku%20eq%20'MCN-GAAUAS-34OZ-CLR-2'%20or%20Sku%20eq%20'MCN-IMFRKSDWR-3PC-10X6-BLCK-2'%20or%20Sku%20eq%20'HGR-SSACSBC-SML-BLCK-2'%20or%20Sku%20eq%20'MCN-MGAA8OZWGRA-8OZ-CLR'%20or%20Sku%20eq%20'HGR-SSACSBC-SML-BLCK-HPN'%20or%20Sku%20eq%20'UPD-JOJOFLEECETHROW-PNK-HPNR'%20or%20Sku%20eq%20'MCN-MAURAWGR-34OZ-CLR'%20or%20Sku%20eq%20'HGR-SSACSBC-SML-BLCK'%20or%20Sku%20eq%20'UPD-JOJOFLEECETHROW-PNK-HPNR'%20or%20Sku%20eq%20'MCN-MAURAWGR-34OZ-CLR'%20or%20Sku%20eq%20'UPD-JOJOFLEECETHROW-PNK-HPNR'%20or%20Sku%20eq%20'HGR-SSACSBC-SML-BLCK-HPN'%20or%20Sku%20eq%20'MCN-GATTMRPBAC-6PC-BLK-HVAR-AMZN')`

As you can see, after the opening parenthesis there is an extra `=` in this part `&$filter=(=Sku%20eq`

**Comment by nicam** [comment]
I found a fix:

The error lies in the Grammar.php file
If you change the function whereNested to use a offset of 9 instead of 8 it works:

```    protected function whereNested(Builder $query, $where)
    {
        // Here we will calculate what portion of the string we need to remove. If this
        // is a join clause query, we need to remove the "on" portion of the SQL and
        // if it is a normal query we need to take the leading "$filter=" of queries.
        // $offset = $query instanceof JoinClause ? 3 : 6;
        $offset = 9;
        return '('.substr($this->compileWheres($where['query']), $offset).')';
    }
```

**Comment by nicam** [comment]
This is fixed in https://github.com/saintsystems/odata-client-php/pull/24

**Comment by nicam** [comment]
@calebn the fix is merged

**Comment by anderly** [comment]
Fixed in #24.
